internal/common/metrics.go

```diff
@@ -103,11 +103,11 @@ func runMetrics(controlPlaneMetricsSupported bool) {
 	})
 }
 
-func checkMetricPresent(mcClient *client.Client, metric string, prometheusBaseUrl string, testPodName string, testPodNamespace string) func() error {
+func checkMetricPresent(mcClient *client.Client, metric string, mimirUrl string, testPodName string, testPodNamespace string) func() error {
 	return func() error {
 		query := fmt.Sprintf("absent(%[1]s{cluster_id=\"%[2]s\"}) or label_replace(vector(0), \"cluster_id\", \"%[2]s\", \"\", \"\")", metric, state.GetCluster().Name)
 
-		cmd := []string{"wget", "-O-", "-Y", "off", fmt.Sprintf("%[1]s/api/v1/query?query=%[2]s", prometheusBaseUrl, url.QueryEscape(query))}
+		cmd := []string{"wget", "-O-", "-Y", "off", "--header", "\"X-Scope-OrgID: anonymous|giantswarm\"", fmt.Sprintf("%[1]s/api/v1/query?query=%[2]s", mimirUrl, url.QueryEscape(query))}
```

Comment:
I think it should be without quotes, but didn't see it being tried in the previous commits.
```suggestion
		cmd := []string{"wget", "-O-", "-Y", "off", "--header", "X-Scope-OrgID: anonymous|giantswarm", fmt.Sprintf("%[1]s/api/v1/query?query=%[2]s", mimirUrl, url.QueryEscape(query))}
```

Reply:
I swear if it's that i'm gonna cry 🤣